contracts/HolographFactory.sol

```diff
@@ -193,7 +224,7 @@ contract HolographFactory is Admin, Initializable, Holographable, HolographFacto
     DeploymentConfig memory config,
     Verification memory signature,
     address signer
-  ) external {
+  ) public {
```

Comment:
why the change to public visibility?

Reply:
I think it was changed because its now being called from within this contract `deployHolographableContractMultiChain`.